src/components/Tipseen/__stories__/tipseen.stories.scss

```diff
@@ -1,7 +1,19 @@
 @import "../../../styles/themes";
+@import "../../../styles/global-css-settings";
 @import "../../../styles/typography";
 
+$component-rules-grid-gap: $spacing-xxl;
```

Comment:
it's not beautiful I know :(

Reply:
why u need individual variables for that?